ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -17129,9 +17155,12 @@ ABoundActivity * HqlCppTranslator::doBuildActivitySOAP(BuildCtx & ctx, IHqlExpre
     if (action)
         doBuildVarStringFunction(instance->startctx, "getSoapAction", action->queryChild(0));
 
+    doBuildHttpHeaderStringFunction(instance->startctx, expr);
+
     IHqlExpression * httpHeader = expr->queryAttribute(httpHeaderAtom);
     if (httpHeader)
```

Comment:
New workunits don't need to include the old functions - this could be deleted.  (New workunits don't need to run on old systems.)